backends/p4fmt/toP4/toP4.h

```diff
@@ -0,0 +1,266 @@
+#ifndef P4FMT_TOP4_TOP4_H_
+#define P4FMT_TOP4_TOP4_H_
+
+#include "frontends/common/resolveReferences/resolveReferences.h"
+#include "ir/ir.h"
+#include "ir/visitor.h"
+#include "lib/sourceCodeBuilder.h"
+
+namespace P4Fmt {
+namespace P4 {
```

Comment:
I feel it's better not to introduce new nested namespaces unless absolutely necessary. Also there might be ambiguity between `::P4Fmt::P4` and `::P4` in unintentional ways. For now, put all P4 formatter code under `::P4Fmt` should be good enough.